integration-tests/src/test_loop/builder.rs

```diff
@@ -43,7 +44,7 @@ use super::utils::network::{chunk_endorsement_dropper, partial_encoded_chunks_dr
 
 pub(crate) struct TestLoopBuilder {
     test_loop: TestLoopV2,
-    genesis: Option<Genesis>,
+    genesis_and_epoch_config_store: Option<(Genesis, EpochConfigStore)>,
```

Comment:
can we keep these fields separate? (even though same function generates them)?

Reply:
Again, that's more lines of code... I can do that, once the general idea is approved by more reviewers.